docs/usage/models.md

```diff
@@ -495,3 +495,27 @@ _(This script is complete, it should run "as is")_
 
 This is a deliberate decision of *pydantic*, and in general it's the most useful approach. See 
 [here](https://github.com/samuelcolvin/pydantic/issues/578) for a longer discussion on the subject.
+
+
+## Model signature
+
+Any *pydantic* model have its own signature generated based on its fields:
+
+```py
+{!.tmp_examples/models_signature.py!}
+```
+
+Accurate signature is useful for introspection purposes and libraries like `FastAPI` or `hypothesis-auto`.
```

Comment:
The signature introspection actually lives in upstream in `hypothesis` (in the `builds()` and `from_type()` strategies) - `hypothesis-auto` just wraps them.

Reply:
Thank you, will correct. Or maybe, you could suggest the change here and I'll commit it, if you want